src/machine/mod.rs

```diff
@@ -196,8 +196,11 @@ pub trait SupportMachine: CoreMachine {
         for value in values.iter().rev() {
             let address =
                 self.registers()[SP].overflowing_sub(&Self::REG::from_u8(Self::REG::BITS / 8));
-
-            self.memory_mut().store32(&address, value)?;
+            if Self::REG::BITS == 64 {
```

Comment:
Can we enable this only in VERSION1, so as to preserve compatibility?

Reply:
Got!